{"name": "test_login_1_success[success_data0]", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <TestCases.Test0LoginCases.test_00_login.TestLogin object at 0x00000285D93EC820>\naccess_web = (<selenium.webdriver.chrome.webdriver.WebDriver (session=\"3b465e052d0e2096450d6eb08db98bcb\")>, <PageObjects.login_page.LoginPage object at 0x00000285D948FFD0>)\nsuccess_data = {'password': 'test123456', 'title': '正常登录', 'user': '17307428595'}\n\n    @pytest.mark.parametrize(\"success_data\", LD.success_data)\n    @pytest.mark.smoke3\n    def test_login_1_success(self, access_web, success_data):  # fixture的函数名称作为用例参数，用来接收fixture的返回值\n        my_log.info(\"****正常用例——{}****\".format(success_data['title']))\n        # 步骤 输入用户名：Xxx 密码：XXX 点击登录\n        access_web[1].login(success_data['user'], success_data['password'])\n        # 断言 首页中能否找到退出元素\n        try:\n            with allure.step(\"step6：断言是否存在退出链接\"):\n                # assert IndexPage(access_web[0]).isExist_logout_ele()\n>               assert False\nE               assert False\n\nTestCases\\Test0LoginCases\\test_00_login.py:98: AssertionError"}, "steps": [{"name": "step1：等待登录按钮可见", "status": "passed", "start": 1646229702633, "stop": 1646229702668}, {"name": "step2：在('xpath', '//input[@name=\"phone\"]')中输入用户名：17307428595", "status": "passed", "start": 1646229702668, "stop": 1646229702818}, {"name": "step3：在('xpath', '//input[@name=\"password\"]')中输入密码：test123456", "status": "passed", "start": 1646229702818, "stop": 1646229702961}, {"name": "step4：勾选记住密码", "status": "passed", "start": 1646229702961, "stop": 1646229702961}, {"name": "step5：点击登录按钮", "status": "passed", "start": 1646229702961, "stop": 1646229703035}, {"name": "step6：断言是否存在退出链接", "status": "failed", "statusDetails": {"message": "AssertionError: assert False\n", "trace": "  File \"D:\\JetBrains\\PyCharm\\PycharmProjects\\Web_Framework_V2\\TestCases\\Test0LoginCases\\test_00_login.py\", line 98, in test_login_1_success\n    assert False\n"}, "start": 1646229703035, "stop": 1646229703035}, {"name": "添加失败截图...", "status": "passed", "attachments": [{"name": "失败截图", "source": "ca05ca95-eaad-48e6-804a-1e14f6ce681b-attachment.png", "type": "image/png"}], "start": 1646229703258, "stop": 1646229703943}], "attachments": [{"name": "log", "source": "d54c572c-ca4d-4834-85de-f1c86f19c53f-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "success_data", "value": "{'user': '17307428595', 'password': 'test123456', 'title': '正常登录'}"}], "start": 1646229702631, "stop": 1646229703038, "uuid": "1d877d00-b54a-4795-b67f-653d9f371209", "historyId": "375ca11fda12ee44766fa033dee5124e", "testCaseId": "a09205c33e5357083a30ed3b570e3817", "fullName": "TestCases.Test0LoginCases.test_00_login.TestLogin#test_login_1_success", "labels": [{"name": "tag", "value": "smoke3"}, {"name": "tag", "value": "@pytest.mark.usefixtures('refresh_page')"}, {"name": "parentSuite", "value": "TestCases.Test0LoginCases"}, {"name": "suite", "value": "test_00_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-KL6VQQI"}, {"name": "thread", "value": "17232-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.Test0LoginCases.test_00_login"}]}